cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-connect.php

```diff
@@ -313,6 +311,47 @@ function ( $a ) {
 		return $result;
 	}
 
+	/**
+	 * Extracts the CNAME from a parsed connection URL.
+	 *
+	 * @param array $parsed_url
+	 * 
+	 * @return string|null
+	 */
+	protected function extract_cname( $parsed_url ) {
+		$cname = null;
+
+		if ( ! empty( $test['query'] ) ) {
+			$config_params = array();
+			wp_parse_str( $parsed_url['query'], $config_params );
+			$cname = isset( $config_params['cname'] ) ? $config_params['cname'] : $cname;
+		} else if ( ! empty( $parsed_url['path'] ) ) {
```

Comment:
Can we keep `elseif` for consistency.
```suggestion
		} elseif ( ! empty( $parsed_url['path'] ) ) {
```